src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -354,6 +354,11 @@ private List<OpenApiParameter> GetOpenApiParameters(MethodInfo methodInfo, Endpo
 
         foreach (var parameter in parameters)
         {
+            if (parameter.Name is null)
```

Comment:
Do we need add a test for this?

Reply:
Per offline discussion, added parameter name validation to the `Flatten` method too.